components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -100,6 +100,7 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
                 channelIdsToAdd: [this.props.channel.id],
             },
         });
+        trackEvent('ui', 'ui_sidebar_channel_menu_createCategory');
```

Comment:
Other places we have the event name as createCategory so went with it but this could also be written as `ui_sidebar_channel_menu_moveToNewCategory` i am 0/5 so if others have an opinion. Do let me know

Reply:
I like `ui_sidebar_channel_menu_moveToNewCategory` a bit better now that you've mentioned it. I don't imagine we'd add a standalone `Create Category` option to the channel menu, but naming this `moveToNewCategory` does help the event name stand out.

Reply:
@esethna Do you have any opinions on these event names? We have:
- `ui_sidebar_menu_create_category` for creating a category from the plus button
- `ui_sidebar_category_menu_createCategory` for creating a category from a category's dot menu
- Either `ui_sidebar_channel_menu_createCategory` or `ui_sidebar_channel_menu_moveToNewCategory` for creating a category from a channel's Move To option

Reply:
ui_sidebar_menu_create_category > ui_sidebar_menu_createCategory

Reply:
Id prefer ui_sidebar_channel_menu_createCategory so it's consistent with the other createCategory labels

Reply:
Okay, the 3 events are now
- ui_sidebar_category_menu_createCategory
- ui_sidebar_menu_createCategory
- ui_sidebar_channel_menu_createCategory